test/controllers/referral.controller.test.js

```diff
@@ -655,15 +656,26 @@ describe('Referral Controller', () => {
   });
 
   describe('Reward a referral', () => {
+    const method = 'put';
+    const endpoint = '/api/v1/referral/rewarded';
```

Comment:
Lets use present tense here to make it more understandable that we are just rewarding the referral. Using rewarded makes it sound like we have previously rewarded the referral

```suggestion
    const endpoint = '/api/v1/referral/reward';
```